mvpconductor/src/main/java/com/harshitbangar/mvpconductor/BaseController.java

```diff
@@ -24,32 +25,38 @@ public BaseController(@Nullable Bundle args) {
 
   @NonNull @Override
   protected final View onCreateView(@NonNull LayoutInflater inflater, @NonNull ViewGroup container) {
```

Comment:
This can return U since java supports covariant overriding - http://javarevisited.blogspot.in/2014/03/covariant-method-overriding-of-java-5.html. At bytecode level, it is going to generate an extra method with the correct cast. If it is going to be used a lot it should return U instead of V then.